Core/Security/ProxyInfo.py

```diff
@@ -156,8 +156,6 @@ def formatProxyStepsInfoAsString(infoList):
                 'group', 'VOMS ext', 'lifetime', 'extensions'):
       if key in stepInfo:
         value = stepInfo[key]
-        if key == 'serial':
-          value = base64.b16encode(value)
```

Comment:
Why removing that ?

Reply:
This lines raise error:

```
Traceback (most recent call last):
  File "/opt/dirac/pro/DIRAC/FrameworkSystem/scripts/dirac-proxy-info.py", line 115, in <module>
    gLogger.notice(formatProxyStepsInfoAsString(stepInfo))
  File "/opt/dirac/pro/DIRAC/Core/Security/ProxyInfo.py", line 162, in formatProxyStepsInfoAsString
    value = base64.b16encode(value)
  File "/opt/dirac/pro/diracos/usr/lib64/python2.7/base64.py", line 263, in b16encode
    return binascii.hexlify(s).upper()
TypeError: b2a_hex() argument 1 must be string or buffer, not long
```

Reply:
But why ? That is also new. This code has been like that for ever, so why is it now receiving a long ?

Reply:
This error disappeared during reinstallation, so I return this lines.

Reply:
Should this change be removed then?

Reply:
IIUC this method has been use only in `dirac-proxy-info` to print list of Proxy steps dictionaries as string. IMHO if we assume there may be a raise exception here and it's ok in output instead of doing error log in this place, then should to remove.